.jenkins/jenkins.sh

```diff
@@ -55,18 +55,18 @@ fi
 action="$1"
 backend="$input_backend"
 experiment="$3"
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
Probably "I like using allcaps so I made my new varname allcaps but I kept the semi-existing varname lowcaps because someone else wrote it that way". I'll grep and replace these.

Reply:
After I noticed the first jenkins.sh file has

```export JENKINS_TAG=${JENKINS_TAG//[,=\/]/-}
if [ ${#JENKINS_TAG} -gt 85 ]; then
	NAME=`echo ${JENKINS_TAG} | md5sum | cut -f1 -d" "`
	export JENKINS_TAG=${NAME//[,=\/]/-}-${BUILD_NUMBER}
fi
echo "JENKINS TAG ${JENKINS_TAG}"

if [ -z ${VIRTUALENV} ]; then
    echo "setting VIRTUALENV"
    export VIRTUALENV=${SCRIPT_DIR}/../venv_${JENKINS_TAG}
fi
```
And that there are over 20 references to SCRIPT_DIR, many of them not from this PR, I'd like to leave this as-is if that's OK.

Reply:
so Google's [styleguide](https://google.github.io/styleguide/shellguide.html#variable-names) says 
> **Variable Names:** Lower-case, with underscores to separate words. Ex: `my_variable_name`
    **Constants** and **Environment Variable Names**: All caps, separated with underscores, declared at the top of the file if possible. Ex: `MY_CONSTANT`


We don't have to strictly obey this convention but I like to follow it when possible. I would more so argue that the buildenv_loc should also be a constant?

Reply:
OK, I can update buildenv_loc.